www/src/components/account/billing/BillingSubscriptionProvider.tsx

```diff
@@ -0,0 +1,67 @@
+import { ReactNode, useContext, useMemo } from 'react'
+import { useQuery } from '@apollo/client'
+import moment from 'moment'
+import posthog from 'posthog-js'
+
+import SubscriptionContext, { SubscriptionContextType } from '../../../contexts/SubscriptionContext'
+import { PlatformSubscription } from '../../../generated/graphql'
+import PlatformPlansContext from '../../../contexts/PlatformPlansContext'
+
+import BillingError from './BillingError'
+import BillingLoading from './BillingLoading'
+import { SUBSCRIPTION_QUERY } from './queries'
+
+type BillingSubscriptionProviderPropsType = {
+  children: ReactNode
+}
+
+function BillingSubscriptionProvider({ children }: BillingSubscriptionProviderPropsType) {
+  const {
+    data,
+    loading,
+    error,
+    refetch,
+  } = useQuery(SUBSCRIPTION_QUERY, { fetchPolicy: 'network-only' })
+  const { proPlatformPlan, proYearlyPlatformPlan, enterprisePlatformPlan } = useContext(PlatformPlansContext)
+
+  const pricingFeaturesEnabled = useMemo(() => posthog.isFeatureEnabled('pricing'), [])
```

Comment:
Did we switch from growthbook to posthog for feature flagging?

Reply:
Yep, we are testing that now.